stack/indexer/rpm/wazuh-indexer.spec

```diff
@@ -170,6 +170,18 @@ if [ -f /etc/os-release ]; then
   fi
 fi
 
+# If is an upgrade, move the securityconfig files if they exist (4.3.x versions)
+if [ $1 = 2 ]; then
```

Comment:
Use ${} for the variables, please check the rest of the script.

Reply:
Done.